client/lib/scroll-to/test/index.js

```diff
@@ -21,8 +15,8 @@ describe( 'scroll-to', () => {
 				y: 300,
 				duration: 1,
 				onComplete: () => {
-					expect( window.scrollTo.lastCall.args[ 0 ] ).toEqual( 500 );
-					expect( window.scrollTo.lastCall.args[ 1 ] ).toEqual( 300 );
+					expect( window.scrollTo.mock.lastCall[ 0 ] ).toEqual( 500 );
+					expect( window.scrollTo.mock.lastCall[ 1 ] ).toEqual( 300 );
```

Comment:
Could it be `.toHaveBeenCalledWith( 500, 300 )` or are there any extra args that we're not checking?